CMSgov/easi-app
# Change Contract Vehicle to Contract Number

# EASI-1977

## Changes and Description

This incorporates both the frontend and backend work for EASI-1977, as well as the work for [EASI-2229](https://jiraent.cms.gov/browse/EASI-2229) laying some of the ground work for linking system intakes with contracts/systems.

Previous PRs:
- Frontend: https://github.com/CMSgov/easi-app/pull/1752
- Backend: https://github.com/CMSgov/easi-app/pull/1747
- EASI-2229 work: https://github.com/CMSgov/easi-app/pull/1741

Once this is merged and the deployed, the following changes will **go live**:
- When creating an intake, users will be asked for "Contract number" instead of "Contract vehicle".
- When viewing an already-submitted intake with a contract number, only the contract number will be displayed.
- When viewing an (older) already-submitted intake with a contract vehicle, the vehicle will be displayed, and "Not Entered" will be displayed as the contract number.

## How to test this change

Check against acceptance criteria in Jira (https://jiraent.cms.gov/browse/EASI-1977).
